src/main/pg/rx.h

```diff
@@ -24,7 +24,7 @@
 
 #include "pg/pg.h"
 
-#define GET_FRAME_ERR_LPF_FREQUENCY(period) (1 / (period / 10.0f))
+#define GET_FRAME_ERR_LPF_FREQUENCY(period) (1.0f / ((period) * 0.1f/*[s]*/))
```

Comment:
The comment in between the letters is a bit hard on the eyes, imo
```suggestion
#define GET_FRAME_ERR_LPF_FREQUENCY(period) (10.0f / period)  // period in deciseconds (0.1s)
```

Reply:
I have to say I had to state at that myself for a bit. I like the suggestion.